packages/fontconfig.rb

```diff
@@ -79,6 +79,14 @@ def self.install
     File.write("#{CREW_DEST_PREFIX}/etc/env.d/fontconfig", @env)
   end
 
+  def self.preinstall
+    @device = JSON.parse(File.read("#{CREW_CONFIG_PATH}device.json"), symbolize_names: true)
+    if @device[:installed_packages].any? { |elem| elem[:name] == 'freetype' }
+      system "sed -i '/*freetype*/d' filelist"
+      system "sed -i '/*freetype*/d' dlist"
+    end
+  end
```

Comment:
Does it make sense to put this above `def self.install` to better represent the progression/flow of the packaging process?

Reply:
Yes and no?

Yes, because it happens _before_ the actual install (for which `def self.install` is irrelevant.)
No, because it happens _after_ the packaging process which uses `def self.install`.

(This is another argument I suppose for renaming `def self.install` def self.package`)